overlord/managers_test.go

```diff
@@ -1598,6 +1598,33 @@ type: os
 
 }
 
+func (s *mgrsSuite) mockSuccessfulReboot(c *C, bloader *bootloadertest.MockBootloader) {
+	st := s.o.State()
+	restarting, _ := st.Restarting()
+	c.Check(restarting, Equals, true, Commentf("mockSuccessfulReboot called when there was no pending restart"))
+	state.MockRestarting(st, state.RestartUnset)
+	err := bloader.SetTryingDuringReboot()
+	c.Assert(err, IsNil)
+	s.o.DeviceManager().ResetBootOk()
+	st.Unlock()
+	defer st.Lock()
+	err = s.o.DeviceManager().Ensure()
+	c.Assert(err, IsNil)
+}
+
+func (s *mgrsSuite) mockRollbackAccrossReboot(c *C, bloader *bootloadertest.MockBootloader) {
+	st := s.o.State()
+	restarting, _ := st.Restarting()
+	c.Check(restarting, Equals, true, Commentf("mockRollbackAccrossReboot called when there was no pending restart"))
+	state.MockRestarting(st, state.RestartUnset)
+	err := bloader.SetRollbackAcrossReboot()
+	c.Assert(err, IsNil)
+	s.o.DeviceManager().ResetBootOk()
+	st.Unlock()
+	s.o.Settle(settleTimeout)
+	st.Lock()
+}
+
 func (s *mgrsSuite) TestInstallKernelSnapUpdatesBootloader(c *C) {
```

Comment:
now that we have gadget assets updates this test name is slightly misleading, maybe  UpdatesBootloaderEnv ?  and we should look if there are more test with such names...